quipucords/api/inspectresult/model.py

```diff
@@ -92,6 +92,8 @@ def default(self, o):
             return o.dict()
         if isinstance(o, datetime):
             return o.isoformat()
+        if isinstance(o, set):
+            return sorted(o)
```

Comment:
Just curious: Why are we sorting sets? Just for readability by human eyes and diff-ability?

Reply:
None.
Json encoder doesn't know how to serialize sets! 😰 😅

Reply:
IT DOESN'T? 😱 

Jeez. 🤦